### DISCUSSION TITLE
Actions after selection

### DISCUSSION THREAD

**Discussion by kariha** [easepick/easepick#44] [Q&A]
When the selection process is completed, I want to print the dates in 2 input fields in the format I want.

Also, when the selection is done, I need to post the dates to an api json file and do some javascript operations.

Thanks for your help.

**Comment by wakirin** [comment]
I see that people don't want to read the docs.
So, now we will use some rules from Stack Overflow.
#45

**Comment by kariha** [comment]
I'm not very good at javascript. I have read the documentation on easepick.com. but I couldn't see any document for the operations after selecting the date.

If there is a document page about the events, please send it to me, I would be very grateful. I could not find.

**Reply by wakirin** [reply]
https://easepick.com/packages/core#event-select

```js
picker.on('select', (e) => {
   // e.detail contains information about select
   // for single picker will be date
   // for range picker will start and date
   console.log(e.detail); 
});
```
